model/project_parser_merge_functions.go

```diff
@@ -23,10 +23,10 @@ func (pp *ParserProject) mergeUnorderedUnique(toMerge *ParserProject) error {
 	for _, task := range toMerge.Tasks {
 		if _, ok := taskNameExist[task.Name]; ok {
 			catcher.Errorf("task '%s' has been declared already", task.Name)
-		} else {
-			pp.Tasks = append(pp.Tasks, task)
-			taskNameExist[task.Name] = true
+			continue
```

Comment:
Thanks for change the other if-else uses to be consistent as well! 💯 (no change)